manuscript/draft.Rmd

```diff
@@ -81,53 +83,57 @@ in the future.
       won't fit well.
-    * For this reason (and because the time series methods assumed 
-      Gaussian errors), we defined model performance throughout this paper in 
+    * For this reason (and because the time series methods assumed
+      Gaussian errors), we defined model performance throughout this paper in
       terms of Gaussians (e.g. mean squared error), rather than with count
       distributions.
         * Some of the modeling approaches (SDMs and JSDMs) did use discrete
           distributions internally, however.
 * Splitting the data temporally
-  * Our data set began in `r settings$start_yr`, which was the first year that 
+  * Our data set began in `r settings$start_yr`, which was the first year that
     NDVI data was available from GIMMS
-  * All the models were trained on the data that was collected through 
+  * All the models were trained on the data that was collected through
     `r settings$last_train_year`
-  * Sites were included in the analysis if there were at least 
+  * Sites were included in the analysis if there were at least
     `r settings$min_num_yrs` years of data available during this period.
-  * The models were tested/evaluated on the data from 
-    `r settings$last_train_year + 1` `r settings$end_yr`.
+  * The models were tested/evaluated on the data from
+    `r settings$last_train_year + 1` `r settings$end_yr`.  
+
+  Bird species richness was compiled from the Breeding Bird Survey (BBS) [@pardieck2017] obtained using the Data Retriever software [@morris203]. The BBS data was initially filtered to exclude all nocturnal, cepuscular, and aquatic species (since these species are not well sampled by BBS methods), as well as unidentified species, and hybrids. All data from routes that did not meet BBS acceptable criteria were excluded. Only routes within the continental U.S.A were used so as to match the climatic dataset.  
+  We used the 4 km resolution PRISM data [@prism] to calculate eight bioclimatic variables identified as relevant to bird distributions [@harris2015] for each year in the dataset: mean diurnal range, isothermality, max temperature of the warmest month, mean temperature of the wettest quarter, mean temperature of the driest quarter, precipitation seasonality, precipitation of the wettest quarter, precipitation of the warmest quarter. Satellite derived NDVI with an 8 km resolution was from the NDIV3g dataset [@pinzon2014] and was available from the time period 1981-2013. Elevation was from the SRTM 90m elevation dataset [@jarvis2008] obtained using the R package raster [@hijmans2014]. BBS route starting point coordinates were used to extract all environmental data. Climatic variables and NDVI were aggregated to the average values within a 40 km radius of the start point, equal to the length of a BBS route. NDVI was also subset to average summer (May, June, July) and winter (December, January, Feburary) values.
```

Comment:
If I remember correctly, we had a rule for shifting the Fall climate data back one year. For instance, Fall 1998 happened after the 1998 transect, so we used Fall 1997. If that's correct, I think we should have a sentence about it.